modules/monitoring-and-analyzing-the-health-of-your-clusters.adoc

```diff
@@ -15,9 +19,9 @@ Wait until the Compliance Operator returns the scan results. It might take a few
 
 .Procedure
 
-. In the {product-title-short} portal, go to the *Compliance (2.0) -> Cluster Compliance -> Coverage* tab.
+. In the {product-title-short} portal, go to *Compliance -> Coverage*.
```

Comment:
This is where we need UI/UX to chime in.  There were significant changes to the screens for this section.  We need to figure out a way to incorporate those to make sure the information is complete and the current information is not obsolete.

Reply:
I think the information here along with what's up in "Coverage page overview" section is accurate, but I'm not sure I'd consider it complete. How detailed should these sections be? I think the following should be mentioned in some way or another:

- When viewing the coverage page, if you apply a filter on a schedule, all results will be filtered accordingly. This filter remains active across all coverage pages until it is cleared.
- Results are always based on a single profile. Profiles are grouped by their associated benchmarks and can be selected using the toggle group.
- Compliance percentage is calculated based on the number of passing checks over the total number of checks

Reply:
I like these recommendations @bradr5